cctk/ensemble.py

```diff
@@ -113,7 +113,7 @@ def molecule_list(self):
 
     def properties_list(self):
         """
-        Returns a list of the constituent molecules.
+        Returns a list of dictionaries. One dictionary per geometry. Each dictionary contains the property names and property values for each geometry in the ensemble. 
```

Comment:
This looks to have been a copy/paste bug in the original.  Good job catching it.